### PR TITLE
Clarify values file wording in the deployment section

### LINKED ISSUE #952: Clarify "Deployment Configuration" section for all platforms
Example: https://documentation.suse.com/suse-cap/2.0/single-html/cap-guides/#sec-cap-aks-config

`provides a complete example deployment configuration.`
This should say that it's a minimal rather than a complete config. The minimal config can then be used as a starting point to add other helm values for optional features.

`The format of the kubecf-config-values.yaml file has been restructured completely. Do not re-use the previous version of the file. Instead, source the default file from the appendix in Section A.1, “Complete suse/kubecf values.yaml File”. `
Need to be clear what we mean by previous version (e.g. the CAP 1.x style file)

We may need to reconsider pointing to the appendix (with the overwhelming amount of listed values) since some users may use that as their basis.

### PR DESCRIPTION
**Pull request by KucharczykL**
Closes #952.

### CODE REVIEW

File: xml/repeated-content-decl.ent [hunk 1]
```diff
@@ -1989,8 +1989,9 @@ User:           username@ldap.example.com
 
 <!ENTITY values-file-changes
 '<para xmlns="http://docbook.org/ns/docbook" xmlns:xlink="http://www.w3.org/1999/xlink">
-  The format of the &values-filename; file has been restructured completely.
-  Do not re-use the previous version of the file. Instead, source the default file from the appendix in
+  The format of the &values-filename; file has been restructured completely in
+  &cap; 2.0. Do not re-use the &cap; 1.x version of the file. Instead, source the
```
Comment (btat) [inline xml/repeated-content-decl.ent:1993]: For the first sentence, any opinions on changing it to `2.x` since it's applicable to more than just 2.0 now?

For the second sentence, could you rephrase the part about sourcing the file. We want to avoid possible misunderstandings where people are just copy and pasting the whole thing and being overwhelmed.

**Review by btat — CHANGES_REQUESTED** [review]
Right now, the EKS and CaaSP chapters have a different intro paragraph for the `Deployment Configuration` section. Could you unify the paragraph for all 4 platforms as well.

File: xml/repeated-content-decl.ent (hunk 1 above)
Comment (KucharczykL) [reply xml/repeated-content-decl.ent]: I have updated the wording. Let me know if it still needs improvement.

**Review by btat — APPROVED** [review]
